crates/optimism/trie/src/provider.rs

```diff
@@ -207,3 +207,27 @@ impl<'a, Storage: OpProofsStore> BytecodeReader for OpProofsStateProviderRef<'a,
         self.latest.bytecode_by_hash(code_hash)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::InMemoryProofsStorage;
+    use reth_provider::noop::NoopProvider;
+
+    #[test]
+    fn test_op_proofs_state_provider_ref_debug() {
+        let latest: Box<dyn StateProvider> = Box::new(NoopProvider::default());
+        let storage: crate::OpProofsStorage<InMemoryProofsStorage> =
+            InMemoryProofsStorage::new().into();
+        let block_number = 42u64;
+
+        let provider = OpProofsStateProviderRef::new(latest, &storage, block_number);
+
+        let debug_output = format!("{:?}", provider);
+
+        assert!(debug_output.contains("OpProofsStateProviderRef"));
+        assert!(debug_output.contains("storage"));
+        assert!(debug_output.contains("block_number"));
+        assert!(debug_output.contains("42"));
```

Comment:
let's just check the whole string at once so we can check the formatting which is what we want
```suggestion
        assert_equals!(debug_output, "OpProofsStateProviderRef { .. }"));
```
`{ .. }` needs to be filled in here